test/helpers/credential/www-Authenticate.ts

```diff
@@ -0,0 +1,7 @@
+export function wwwAuthenticateHeaderContainsCorrectError(
+  header: string,
+): boolean {
+  if (!header.startsWith("Bearer")) return false;
+
+  return header.includes('error="invalid_token"');
```

Comment:
I know I said I'd rather not use regular expressions, but I'm worried that doing `header.includes('error="invalid_token"')` returns true for a value such as `'Bearer blaherror="invalid_token"'` and that's wrong.

```suggestion
  return /\berror="invalid_token"/.test(header);
```

Reply:
I know it doesn't matter too much but we need to make sure the string contains `error="invalid_token"`.